sdk/typescript/dev/node/src/commands.ts

```diff
@@ -0,0 +1,31 @@
+import { Container, object, func } from "@dagger.io/dagger"
+
+@object()
+export class Commands {
+  // Container to apply commands on
+  ctr: Container
+
+  constructor(ctr: Container) {
+    this.ctr = ctr
+  }
```

Comment:
Isn't this redundant?

Reply:
What do you recommand? I need to store the container somewhere to apply command on it?

The other way would be to remove `Commands` and do everything at the top level object.

Reply:
Oh, I guess you made it required in this PR to explicitly create a `constructor` function, otherwise no default one will be registered?